### PR TITLE
Add r-skpr

### PR DESCRIPTION
**Pull request by comatrion**
@conda-forge/help-r here is a new package `skpr` from CRAN.
I have used the helper script from https://github.com/bgruening/conda_r_skeleton_helper

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by comatrion** [comment]
Hi @conda-forge/staged-recipes please review my PR!

**Comment by scopatz** [comment]
CC @conda-forge/staged-recipes

File: recipes/r-skpr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    - r-promises
+    - r-rintrojs
+    - r-shiny
+    - r-shinyjs
+    - r-shinythemes
+    - r-survival
+    - r-viridis
+  run:
+    - openblas   # [not win]
+    - mkl        # [win]
```
Comment (isuruf) [inline recipes/r-skpr/meta.yaml]: Remove the two line above

Comment (comatrion) [reply recipes/r-skpr/meta.yaml]: OK thanks.

**Comment by comatrion** [comment]
CC @conda-forge/staged-recipes  Comments addressed and still passing.

**Comment by comatrion** [comment]
Touch

**Pull request closed without merging**